edb/lib/schema.edgeql

```diff
@@ -287,7 +297,9 @@ CREATE TYPE schema::Property EXTENDING schema::Pointer;
 
 ALTER TYPE schema::Pointer {
     CREATE LINK source -> schema::Source;
-    CREATE LINK target -> schema::Type;
+    CREATE LINK target -> schema::Type {
+        ON TARGET DELETE DEFERRED RESTRICT;
```

Comment:
Good call.  I wonder if we can actually turn some `ALLOW` in `structure.py` into this?